example/cumulus-tf/cnm_response_task.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  cnm_response_version  = "2.0.0"
+  cnm_response_version  = "2.0.1-alpha.2.0.1-SNAPSHOT"
```

Comment:
They don't have a full release?

Reply:
I don't believe so: https://github.com/podaac/cumulus-cnm-response-task/releases

Right now, releases 2.0.0+ are just for our integration tests, since they are updated for compatibility with the file schema changes that we haven't released yet:

https://github.com/podaac/cumulus-cnm-response-task/releases/tag/v2.0.0

So it makes some sense that they didn't do a "real" release of 2.0.1 with the log4j patched versions yet

Reply:
Is there one for CNM to granule?? Or should we be using https://github.com/podaac/cumulus-cnm-to-granule/releases/tag/v1.5.2